x-pack/plugins/security_solution/server/lib/telemetry/filterlists/endpoint_alerts.ts

```diff
@@ -102,6 +102,7 @@ const allowlistBaseEventFields: AllowlistFields = {
   user: {
     id: true,
   },
+  Persistence: true,
```

Comment:
Is it under the `event` or the top level? Do you have an example alert you can paste on this PR so I can check/compare?